payments-core/src/main/java/com/stripe/android/model/LinkMode.kt

```diff
@@ -4,6 +4,19 @@ import androidx.annotation.RestrictTo
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 enum class LinkMode(val value: String) {
-    Passthrough("passthrough"),
-    PaymentMethod("payment_method_mode"),
+    Passthrough("PASSTHROUGH"),
+    LinkPaymentMethod("LINK_PAYMENT_METHOD"),
+    LinkCardBrand("LINK_CARD_BRAND"),
 }
+
+/**
+ * Needed for backwards compatibility, as Elements uses these values for
+ * analytics instead of the strings coming from the API.
+ */
+val LinkMode.analyticsValue: String
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    get() = when (this) {
+        LinkMode.Passthrough -> "passthrough"
+        LinkMode.LinkPaymentMethod -> "payment_method_mode"
+        LinkMode.LinkCardBrand -> "link_card_brand"
+    }
```

Comment:
Could this be simplified a bit to:
```suggestion
    get() = this.value.lowercase()
```

sorry if the syntax is slightly off 😅

Reply:
That would result in different values, which will make working with these values in our dashboards complicated. I also want these values to be stable and not be (inadvertently) affected by refactorings.

Reply:
Whoops - I missed that we use a different value for `LinkPaymentMethod` 👍